packages/rs-drive-abci/src/execution/types/quorum.rs

```diff
@@ -12,54 +12,178 @@ use tenderdash_abci::proto::abci::ValidatorSetUpdate;
 use tenderdash_abci::proto::crypto::public_key::Sum::Bls12381;
 use tenderdash_abci::proto::{abci, crypto};
 
-/// Quorum information
-#[derive(Clone, Debug, Serialize, Deserialize)]
-pub struct Quorum {
+/// The validator set is only slightly different from a quorum as it does not contain non valid
+/// members
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct ValidatorSet {
     /// The quorum hash
     pub quorum_hash: QuorumHash,
     /// Active height
     pub core_height: u32,
     /// The list of masternodes
-    pub validator_set: BTreeMap<ProTxHash, Validator>,
+    pub members: BTreeMap<ProTxHash, Validator>,
     /// The threshold quorum public key
     pub threshold_public_key: BlsPublicKey,
 }
 
-impl From<Quorum> for ValidatorSetUpdate {
-    fn from(value: Quorum) -> Self {
-        let Quorum {
+impl ValidatorSet {
+    /// For changes between two validator sets, we take the new (rhs) element if is different
+    /// for every validator
+    #[allow(dead_code)]
```

Comment:
We don't use it?

Reply:
Not for now, but I wanted to leave it in there just in case we needed it.